fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/expressions/LogicalSortExpression.java

```diff
@@ -139,13 +164,13 @@ public int hashCode() {
 
     @Override
     public int hashCodeWithoutChildren() {
-        return Objects.hash(getSortValues(), isReverse());
+        return ordering.hashCode();
```

Comment:
I think with this here you can just delete the `semanticHashCode()` override.

Reply:
Done